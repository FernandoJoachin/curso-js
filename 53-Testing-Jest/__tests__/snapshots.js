const cliente = {
    nombre: 'Fernando',
    balance: 6000,
    tipo: 'VIP'
};

describe('Testing del Cliente', () => {
    test('Validar si es el mismo cliente', () => {
        expect(cliente).toMatchSnapshot();
    });

})