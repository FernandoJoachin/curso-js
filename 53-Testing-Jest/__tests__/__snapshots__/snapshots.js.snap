// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing del Cliente Validar si es el mismo cliente 1`] = `
{
  "balance": 6000,
  "nombre": "Fernando",
  "tipo": "VIP",
}
`;
